io-sim/test/Test/Control/Monad/IOSimPOR.hs

```diff
@@ -1,40 +1,128 @@
+  -- , testProperty "propPermutations" propPermutations
+  , testGroup "IO simulator properties"
+    [ testProperty "read/write graph (IOSim)" (withMaxSuccess 1000 prop_stm_graph_sim)
+    , testProperty "timers (IOSim)"           (withMaxSuccess 1000 prop_timers_ST)
+    , testProperty "timeout (IOSim): no deadlock"
+                                              prop_timeout_no_deadlock_Sim
+    , testProperty "threadId order (IOSim)"   (withMaxSuccess 1000 prop_threadId_order_order_Sim)
+    , testProperty "forkIO order (IOSim)"     (withMaxSuccess 1000 prop_fork_order_ST)
+    , testGroup "throw/catch unit tests"
+      [ testProperty "0" unit_catch_0
+      , testProperty "1" unit_catch_1
+      , testProperty "2" unit_catch_2
+      , testProperty "3" unit_catch_3
+      , testProperty "4" unit_catch_4
+      , testProperty "5" unit_catch_5
+      , testProperty "6" unit_catch_6
+      ]
+    , testGroup "masking state"
+      [ testProperty "set (IOSim)"
+      $ forall_masking_states unit_set_masking_state_ST
+
+      , testProperty "unmask (IOSim)"
+      $ forall_masking_states $ \ms  ->
+        forall_masking_states $ \ms' -> unit_unmask_ST ms ms'
+
+      , testProperty "fork (IOSim)"
+      $ forall_masking_states unit_fork_masking_state_ST
+
+      , testProperty "fork unmask (IOSim)"
+      $ forall_masking_states $ \ms  ->
+        forall_masking_states $ \ms' -> unit_fork_unmask_ST ms ms'
+
+      , testProperty "catch (IOSim)"
+      $ forall_masking_states unit_catch_throwIO_masking_state_ST
+
+      , testProperty "catch: throwTo (IOSim)"
+      $ forall_masking_states unit_catch_throwTo_masking_state_ST
+
+      , testProperty "catch: throwTo async (IOSim)"
+      $ forall_masking_states unit_catch_throwTo_masking_state_async_ST
+
+      , testProperty "catch: throwTo async blocking (IOSim)"
+      $ forall_masking_states unit_catch_throwTo_masking_state_async_mayblock_ST
+      ]
+    , testProperty "evaluate unit test" unit_evaluate_0
+    , testGroup "forkIO unit tests"
+      [ testProperty "1" unit_fork_1
+      ]
+    , testGroup "async exception unit tests"
+      [ testProperty "1"  unit_async_1
+      , testProperty "3"  unit_async_3
+      , testProperty "4"  unit_async_4
+      , testProperty "5"  unit_async_5
+      , testProperty "6"  unit_async_6
+      , testProperty "7"  unit_async_7
+      , testProperty "8"  unit_async_8
+      , testProperty "9"  unit_async_9
+      ]
+    , testGroup "STM reference semantics"
+      [ testProperty "Reference vs Sim"   prop_stm_referenceSim
+      ]
+    , testGroup "MonadFix instance"
+      [ testProperty "purity"     prop_mfix_purity
+      , testProperty "purity2"    prop_mfix_purity_2
+      , testProperty "tightening" prop_mfix_left_shrinking
+      , testProperty "lazy"       prop_mfix_lazy
+      ]
+    ]
+  ]
 
 data Step =
     WhenSet Int Int
   | ThrowTo Int
   | Delay Int
   | Timeout TimeoutStep
-  | CheckStatus Int
```

Comment:
Why it's removed in this PR when we just move files around?

Reply:
Good question hmmm I'll look into this

Reply:
Solved!